crates/bevy_ecs/src/query/mod.rs

```diff
@@ -749,4 +749,37 @@ mod tests {
         let _: [&Foo; 1] = q.many([e]);
         let _: &Foo = q.single();
     }
+
+    #[test]
+    fn par_iter_mut_change_detection() {
+        let mut world = World::new();
+        world.spawn((A(1), B(1)));
+
+        fn propagate_system(mut query: Query<(&A, &mut B), Changed<A>>) {
+            query.par_iter_mut().for_each_mut(|(a, mut b)| {
+                b.0 = a.0;
+            });
+        }
+        let mut propagate_system = IntoSystem::into_system(propagate_system);
+        propagate_system.initialize(&mut world);
```

Comment:
Is there any particular reason that you're manually initializing and calling these systems instead of just adding them to a `Schedule`?

Reply:
Not really, I just copied that approach from the previous test in that file.  Should I replace it with

```rust
        let mut schedule = Schedule::new();
        schedule.add_systems((propagate_system, modify_system).chain());
        schedule.run(&mut world);
        world.clear_trackers();
        schedule.run(&mut world);
        world.clear_trackers();
```
?  

(I thought I could put `World::clear_trackers` in the schedule, too, but exclusive systems can't actually update `last_change_tick`, and without that update the test can't catch the bug.)

Reply:
Yeah, I think that looks better.